### ISSUE TITLE
Missing logging in 2.x branch means no logs make it to Graylog

### ISSUE THREAD

**Issue by madmatt** [silverstripe/cwp-core#43]
CWP provides the Graylog service for developers to record errors in. `cwp-core` previously in the 1.x line automatically logged messages into Graylog, but the `2.x` line seems to have lost this.

Here's what I added to a project to bring this back, using the same `SilverStripe_log` facility that I believe was used previously.

In a new file, e.g. `mysite/_config/logging.yml`:
```
SilverStripe\Core\Injector\Injector:
  Psr\Log\LoggerInterface:
    calls:
      SyslogHandler: [ pushHandler, [ %$SyslogHandler ] ]
  SyslogHandler:
    class: Monolog\Handler\SyslogHandler
    constructor:
      - 'SilverStripe_log'
```

This is enough to push log messages through to CWP's Graylog system.

**Comment by robbieaverill** [comment]
Thanks @madmatt, I've tested your fix and it's good. I've adjusted the names used slightly and made a PR for it at #44

**Comment by ScopeyNZ** [comment]
Closed by #44

**Comment by madmatt** [comment]
Thanks for the super-fast fix team!
